tests/input/credential-type-urls-order-2-ok.json

```diff
@@ -0,0 +1,15 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2"
+  ],
+  "type": [
+    "VerifiableCredential",
```

Comment:
Maybe move `VerifiableCredential` to the end of the list? That seems to be the most likely wrong assumption to be made by implementers: that `type[0]` will always === `VerifiableCredential`, but if it MUST be an unordered set...then that value could be anywhere in the array.

Reply:
This is a good idea and that change was made here: https://github.com/w3c/vc-data-model-2.0-test-suite/pull/45/commits/8d09419f75fa042502ba590bf6faaf0aa61d4f4b

in one case `VerifiableCredential` is in the middle in another it is last in the array.